webhook-app/webhook_helper.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helpers for implementing GitHub webhooks."""
+
+from collections import defaultdict
+import hashlib
+import hmac
+import logging
+
+
+def check_signature(secret, header_signature, request_body):
```

Comment:
I'll keep that in mind. For now I'm sure it works because otherwise none of the github notifications will work if I got it wrong (& I got it wrong several times). :P